erc721-meta-transaction/src/config.ts

```diff
@@ -0,0 +1,2 @@
ZG1pbiIsIk1hcmtldCBBUEkiLCJOb3RpZmljYXRpb25zIEFQSSIsIk5GVC1BUEkiLCJCdXNpbmVzcyBQb3J0YWwgVXNlciJdfSwicmVzb3VyY2VfYWNjZXNzIjp7IkFya2FuZU1hcmtldCI6eyJyb2xlcyI6WyJjYW5jZWw6b2ZmZXIiLCJtYW5hZ2U6YWR2ZXJ0aXNlbWVudHMiLCJtYW5hZ2U6cGFnZXMiLCJyZWZyZXNoOmNvbGxlY3Rpb25zIiwidXBkYXRlOm5mdC1jb2xsZWN0aW9uIiwiY3JlYXRlOmJpZCIsImNyZWF0ZTpzYWxlIiwicmV3YXJkOnVzZXIiLCJjcmVhdGU6c3ViLXVzZXIiLCJjcmVhdGU6bmZ0LWNvbGxlY3Rpb24iLCJkZWxldGU6bmZ0LWNvbGxlY3Rpb24iLCJidXk6c2FsZSIsImNyZWF0ZTphdWN0aW9uIiwidGVybWluYXRlOm9mZmVyIiwiY3JlYXRlOmFubm91bmNlbWVudCIsImNyZWF0ZTpyZXdhcmQtY2FtcGFpZ24iLCJjcmVhdGU6c3VidXNlci1hZGRyZXNzIl19LCJyZWFsbS1tYW5hZ2VtZW50Ijp7InJvbGVzIjpbImNyZWF0ZS1pbnZpdGF0aW9uIiwicXVlcnktY2xpZW50cyIsInF1ZXJ5LWdyb3VwcyJdfSwiTkZULUFQSSI6eyJyb2xlcyI6WyJtYW5hZ2U6dG9rZW4tdHlwZXMiLCJ2aWV3OmVudmlyb25tZW50cyIsIm1pbnQ6dG9rZW5zIiwiY3JlYXRlOm1pbnRlci13YWxsZXQiLCJjcmVhdGU6Y29tcGFueS1taW50ZXItd2FsbGV0IiwibWFuYWdlOm1pbnRlci13YWxsZXQiLCJ2aWV3OnRva2VuLXR5cGVzIiwibWFuYWdlOmFwcHMiLCJ2aWV3OnRva2VucyIsImFkbWluOm1vbml0b3Itc3lzdGVtIiwibWFuYWdlOnRva2VucyIsInZpZXc6aXRlbXMiLCJtYW5hZ2U6Y29udHJhY3RzIiwidXBsb2FkOm1lZGlhIl19LCJOb3RpZmljYXRpb25zIjp7InJvbGVzIjpbInNlbmQ6ZW1haWxzIiwibWFuYWdlOndlYmhvb2tzIl19LCJCdXNpbmVzc1BvcnRhbCI6eyJyb2xlcyI6WyJtYW5hZ2U6d2lkZ2V0LWNsaWVudCIsIm1hbmFnZTpjb21wYW55IiwibWFuYWdlOmFwaS1jbGllbnQiLCJtYW5hZ2U6aW52aXRhdGlvbnMiLCJtYW5hZ2U6dXNlcnMiLCJ2aWV3OmFwaS1jbGllbnQiLCJ2aWV3Omludm9pY2VzIiwibWFuYWdlOndlYmhvb2tzIiwidmlldzpjb21wYW55IiwidmlldzpwYXltZW50LW1ldGhvZHMiLCJ2aWV3OndpZGdldC1jbGllbnQiLCJjcmVhdGU6Y29tcGFueSIsInZpZXc6c2hvcGlmeS1jbGllbnQiLCJ2aWV3OnVzZXJzIiwibWFuYWdlOmNyZWRlbnRpYWxzIiwidmlldzppbnZpdGF0aW9ucyIsIm1hbmFnZTpzdWJzY3JpcHRpb25zIiwibWFuYWdlOnBheW1lbnRzIiwibWFuYWdlOnBheW1lbnQtbWV0aG9kcyIsInJldHJpZXZlOnVzYWdlcyIsInZpZXc6d2ViaG9va3MiLCJtYW5hZ2U6c2hvcGlmeS1jbGllbnQiXX19LCJzY29wZSI6Im1hbmFnZTp1c2VyLXZlcmlmaWNhdGlvbiBoZXhhZ29uLXdhbGxldDp3aXRoZHJhdyBoZXhhZ29uLXRva2VuOmNyZWF0ZSBleHBvcnQ6d2FsbGV0cyBzaWduOndhbGxldHMgdXNlOmFueS13YWxsZXQgY3J5cHRvMmVtYWlsIHZpZXc6d2FsbGV0LWFuYWx5dGljcyBoZXhhZ29uLXdhbGxldDpjcmVhdGUgc2F2ZTp0cmFuc2FjdGlvbiBoZXhhZ29uLXdlYmhvb2s6Y3JlYXRlIGNyZWF0ZTpjb21wYW55LW1pbnRlci13YWxsZXQgdmlldzp3YWxsZXRzIHdoaXRlbGFiZWwgZGVsZXRlOnVzZXItdmVyaWZpY2F0aW9uIGVtYWlsIHVzZTphbGwtd2FsbGV0cyB2aWV3OnByb2ZpbGUgcmV3YXJkOnVzZXIgc2F2ZTpzaWduYXR1cmUgaGV4YWdvbi13YWxsZXQ6Y3JlYXRlLWNvbGxlY3RvciBwcm9maWxlIiwiY29tcGFuaWVzIjpbeyJpZCI6IjEzMzliOGExLTk1ZDktNDc5Ny1iM2MxLThhYWZkYjJmNDhmZCIsIm5hbWUiOiJBbGVrc04ifV0sImNsaWVudEhvc3QiOiI3OS4xMDYuMTIzLjIwNyIsImNsaWVudElkIjoiYWxla3NuLWNhcHN1bGUiLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsIm5pY2tuYW1lIjoiYWxla3NuIiwicHJlZmVycmVkX3VzZXJuYW1lIjoic2VydmljZS1hY2NvdW50LWFsZWtzbi1jYXBzdWxlIiwiY2xpZW50QWRkcmVzcyI6Ijc5LjEwNi4xMjMuMjA3In0.BBCo6Zfw6xxOnzKR34u5P1Qag0tQvgFsMIUcC61zqVkc6K_VX0Ffvs4AmekvSnGJaxY3i_7kzOcg4KzKVyLGy6UtYML_K4RX0loAApt9m31NtXlpdruhvDRS7WGuVBylwvxT2FGeTZwwi6N_J0OCli3hNPjbe4Cwp-8in_uDjQXLOt1sbnXX0Jpalc_WyvYlAPAtn3rdcLP8d7qYEOIROixxjNStC2OaHKTyABuxklK_6elcS1vPR0KkiSTuELHDMGfAP5mPdpcJNmySYCONyTgIRm0qdrTsIb3VHTBwBfIiWWGeQKbEnDojHJPTHCoiXFDC4faONN7KsmApccKVAA';
```

Comment:
remove (or at least a large part) of the bearer token. So maybe include only the first and last 20 characters or something